conans/client/tools/net.py

```diff
@@ -47,8 +49,14 @@ def ftp_download(ip, filename, login='', password=''):
             pass
 
 
-def download(url, filename, verify=True, out=None, retry=2, retry_wait=5, overwrite=False,
+def download(url, filename, verify=True, out=None, retry=None, retry_wait=None, overwrite=False,
              auth=None, headers=None, requester=None):
+
+    if retry is None:
```

Comment:
why not the default to 2 in the argument?

Reply:
Well, I don't want to repeat the `2` and `5` in the `tools.get`. I think is more clear like this.

Reply:
I think it is to avoid propagating the default values to the upstream function, ``tools.get()`` in this case